daliuge-engine/dlg/apps/pyfunc.py

```diff
@@ -403,9 +400,7 @@ def run(self):
 
         inputs = collections.OrderedDict()
         for uid, drop in self._inputs.items():
-            contents = droputils.allDropContents(drop)
-            # allow for Null DROPs to be passed around
-            inputs[uid] = all_contents(contents) if contents else None
+            inputs[uid] = all_contents(drop)
```

Comment:
This change is needed such that `allDropContents` is not called when `PATH` option is selected